docs/pages/docs/handbook/deploying-with-docker.mdx

```diff
@@ -146,7 +148,7 @@ USER nextjs
 COPY --from=installer /app/apps/web/next.config.js .
 COPY --from=installer /app/apps/web/package.json .
 
-# Automatically leverage output traces to reduce image size
+# Automatically leverage output traces to reduce image size 
```

Comment:
Hmm, this trailing white space seems to be in [the source file](https://github.com/adarshaacharya/turborepo/blob/b06f9084b2170ebe74a4dcb44ccd06fd8e023309/examples/with-docker/apps/web/Dockerfile#L41) as well.